src/Symfony/Component/Form/ChoiceList/Factory/ChoiceListFactoryInterface.php

```diff
@@ -31,9 +32,8 @@ interface ChoiceListFactoryInterface
      * The callable receives the choice as only argument.
      * Null may be passed when the choice list contains the empty value.
      *
-     * @param iterable      $choices The choices
-     * @param callable|null $value   The callable generating the choice
-     *                               values
+     * @param iterable                                   $choices The choices
+     * @param callable|string|PropertyPathInterface|null $value   The callable or path for generating the choice values
```

Comment:
there are implementations that might accept all these values. but it's not required by the interface. should be reverted.

Reply:
I understand your comment. We can broaden the argument types in subclasses, that's valid. On the other hand, we can then technically add a type declaration `?callable` to the interface and the Factory will be unable to pass property paths. So we do type hinting in the interface based on possible implementations...